internal/prompts/prompts.go

```diff
@@ -0,0 +1,15 @@
+package prompts
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+// NewDeleteConfirmation creates a prompt to confirm if the entry should be deleted
+func NewDeleteConfirmation(entry string) *survey.Confirm {
```

Comment:
[q] Does this still come up if we do a --force? Should it - I would say no, but don't feel strongly on this.

Reply:
It doesn't show, if you look at https://github.com/mongodb/mcli/blob/c5c1fbce3240a80401f688e173d9e8c348fd8b74/internal/cli/atlas_whitelist_delete.go#L63-L65 you'll see that we return before this gets called if the person used `--force`
This method is meant to encapsulate the creation of the survey struct, I plan to reuse it as we keep making the same struct for all our "confirm" operation only changing the thing between quotes but I didn't make those changes on this PR

Reply:
This is also a go thing where you create a `NewSomething` method for encapsulating ` &struct{}` operations, also since go doesn't have polymorphism they recommend making names quite explicit, hence the  full "new Delete Confirmation" tho I could make it just `NewDeleteConfirm` to keep it in line with `survey`

Reply:
Excellent, thanks for the details!